presto-hive/src/test/java/io/prestosql/plugin/hive/TestOrcPageSourceMemoryTracking.java

```diff
@@ -749,9 +749,7 @@ public Object getWriteValue()
         {
             StringBuilder builder = new StringBuilder();
             String source = writeValue.get();
-            for (int i = 0; i < counter / step; i++) {
-                builder.append(source);
-            }
+            builder.append(String.valueOf(source).repeat(Math.max(0, counter / step)));
```

Comment:
`source` is already a String

did you mean `firstNonNull(source, "null")`?

Reply:
Thanks, this got added somehow